ibm_catalog.json

```diff
@@ -0,0 +1,106 @@
+{
+  "products": [
+    {
+      "name": "deploy-arch-ibm-event-streams",
+      "label": "Event Streams",
+      "product_kind": "solution",
+      "tags": [
+        "event_streams",
+        "ibm_created",
+        "target_terraform",
+        "terraform",
+        "reference_architecture",
+        "solution"
```

Comment:
I recently found out that there is only a pre-defined set of tags that you can use. Please remove `event_streams` and `reference_architecture`. Please add `support_ibm` and `integration`